demo/index.html

```diff
@@ -6,7 +6,7 @@
 	<link rel="stylesheet" href="style.css" />
 	<link href="https://fonts.googleapis.com/css?family=Inconsolata" rel="stylesheet">
 
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/4.5.6/pixi.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/5.2.0/pixi.js"></script>
```

Comment:
@tleunen After changing the version here the canvas in the demo page no longer extends to the bottom of the screen. I'm not sure exactly why that may be the case or if it's intended, but maybe you can take a look.

Reply:
This is because the api changed.
`PIXI.autoDetectRenderer` takes an option object now.
```
let renderer = PIXI.autoDetectRenderer({
      width: 600,
      height: 3425
    });
```

Reply:
I've fixed it on master already. So you can rebase and only keep the `open` fix here now ;)
Thanks a lot for your contributions tho!